### DISCUSSION TITLE
not closing properly

### DISCUSSION THREAD

**Discussion by DepressedHuman** [ppy/osu#14283] [Q&A]
whenever i open osu! lazer and close it after playing a bit, it goes into the background processes on my pc and doesnt fully close, discord still picks up lazer as if im still playing aswell. ive only noticed this in the most recent update. not the biggest issue in the world so a late response isnt a bother. please let me know if more data is needed.

![image](https://user-images.githubusercontent.com/88918326/129433300-899e0525-1fd4-453c-88c0-0a770aeb48ca.png)

osu! version: 2021.809.0-lazer (latest)

**Comment by peppy** [comment]
When this happens could you please right click the process, create a memory dump and email it to me? pe@ppy.sh (it may contain osu! credentials if you are logged in)
